hana/test/compliance

```diff
@@ -0,0 +1 @@
+../../test
```

Comment:
Why is that required for compliance but not for bookshop?

Reply:
this is a symbolic link inside the `hana/test` folder which makes it look like the root `test` folder is also located on `hana/test/compliance` and the root `test` folder contains `compliance`, `bookshop` and `sflight`. So it is required for `bookshop` as well as this includes the `bookshop`.